modules/objdetect/CMakeLists.txt

```diff
@@ -1,2 +1,8 @@
 set(the_description "Object Detection")
-ocv_define_module(objdetect opencv_core opencv_imgproc WRAP java python js)
+ocv_define_module(objdetect opencv_core opencv_imgproc opencv_calib3d WRAP java python js)
+
+if(WITH_QUIRC)
```

Comment:
HAVE_QUIRC

- `WITH_*` - auto detect build feature, user-defined option
- `HAVE_*` - build feature is available

Reply:
@alalek done.